test-crates/duchess-java-tests/tests/ui/passing_null_values.rs

```diff
@@ -5,6 +5,13 @@ duchess::java_package! {
     package take_null;
 
     public class TakeNull { * }
+    public class TakeNullRecord { * }
```

Comment:
we need to change this test to `//@run` — currently it's not actually being run, just compiled.